test/persistence-hooks.suite.js

```diff
@@ -106,7 +106,11 @@ module.exports = function(dataSource, should, connectorCapabilities) {
         };
         TestModel.find(query, function(err, list) {
           if (err) return done(err);
-          hookMonitor.names.should.eql(['access']);
+          if (dataSource.connector.buildNearFilter)
+            hookMonitor.names.should.eql(['access']);
```

Comment:
I am confused. AFAICT, there is no `TestModel` instance created with `location` set. How comes that a connector is returning an non-empty result then? Is this an interference between tests? In that case I think we should either change the model name to a unique value or call `deleteAll` in a before/beforeEach hook.